azurerm/resource_arm_app_service.go

```diff
@@ -160,6 +160,15 @@ func resourceArmAppService() *schema.Resource {
 							Optional: true,
 							Computed: true,
 						},
+						"scm_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  string(web.ScmTypeNone),
+							ValidateFunc: validation.StringInSlice([]string{
+								string(web.ScmTypeNone),
+								string(web.ScmTypeLocalGit),
+							}, false),
+						},
```

Comment:
so it'd also be good to expose the value of the SCM URL as a computed field - which I believe should be returned from [the `GetSourceControl` API call when this is set to `ScmTypeLocalGit`](https://docs.microsoft.com/en-us/rest/api/appservice/webapps/getsourcecontrol)?